frontend/src/composables/useChainSelection.ts

```diff
@@ -27,7 +27,13 @@ export function useChainSelection(
     Object.keys(chains.value).map((chainId) => getChainSelectorOption(chainId, chains.value)),
   );
   const targetChains = computed(() =>
-    sourceChains.value.filter((chain) => chain?.value !== selectedSourceChain.value?.value),
+    sourceChains.value.filter(
+      (chain) =>
+        !(
+          chain?.value === selectedSourceChain.value?.value &&
+          process.env.NODE_ENV === 'development'
```

Comment:
This means we don't filter the selected source chain from the target chains in development, because there will only be one chain. Maybe the expression is not very expressive?

Reply:
But I  think the expression was wrong anyway. And the chain id should be compared not the object.

Reply:
Haha okay.
But please just for the record: why do we need this filter adaption for `development` anyway?

Reply:
👌🏾

Reply:
In the local test deployment, we do the transfers to the same target as the source chain (Ganache). Without the check, the target chain would get filtered and the selector would be empty.
This reasoning is also in the commit message.

Reply:
Oh okay. Sorry that I missed that. Especially since I put such information in commit messages too. 🙈